sql/synth_recipes.sql

```diff
@@ -2586,38 +2586,38 @@ INSERT INTO `synth_recipes` VALUES (34557,0,0,0,0,24,98,26,0,0,0,4099,4241,798,7
 INSERT INTO `synth_recipes` VALUES (34558,0,0,0,0,0,98,0,0,0,0,4099,4241,3550,3550,3550,0,0,0,0,0,3551,3551,3551,3551,1,1,1,1,'Wyrdweave');
 INSERT INTO `synth_recipes` VALUES (34559,0,0,0,52,0,99,48,0,0,0,4099,4241,664,666,821,828,855,855,1132,1132,14376,14377,14377,14377,1,1,1,1,'Rasetsu Samue');
 INSERT INTO `synth_recipes` VALUES (34560,0,0,0,0,54,99,33,0,0,0,4099,4241,813,813,828,879,2340,2472,2476,0,2453,2454,2454,2454,1,1,1,1,'Cursed Cuffs'); -- wiki, BG, jpwiki, AH checked 24/Nov/2021
--- INSERT INTO `synth_recipes` VALUES (34561,0,0,0,0,0,99,255,0,0,0,4099,4241,821,8751,8751,8976,8988,0,0,0,27991,27992,27992,27992,1,1,1,1,'Revealer\'s Mitts');
+-- INSERT INTO `synth_recipes` VALUES (34561,0,0,0,0,0,99,0,0,0,0,4099,4241,821,8751,8751,8976,8988,0,0,0,27991,27992,27992,27992,1,1,1,1,'Revealer\'s Mitts'); -- 2022.03.06 sub leather level unknown
 INSERT INTO `synth_recipes` VALUES (34562,0,0,0,0,0,99,0,0,0,0,4099,4241,821,823,2166,2166,2287,2304,0,0,15891,15892,15892,15892,1,1,1,1,'Al Zahbi Sash');
--- INSERT INTO `synth_recipes` VALUES (34563,0,0,0,0,0,100,0,0,255,0,4102,4244,760,822,829,1767,1769,2113,2476,0,11310,11311,11311,11311,1,1,1,1,'Argent Coat');
--- INSERT INTO `synth_recipes` VALUES (34564,0,0,0,0,0,100,255,0,0,0,4099,4241,826,8751,8751,8976,8988,0,0,0,27228,27229,27229,27229,1,1,1,1,'Revealer\'s Pants');
+INSERT INTO `synth_recipes` VALUES (34563,0,0,0,0,0,100,41,0,20,0,4102,4244,760,822,829,1767,1769,2113,2476,828,11310,11311,11311,11311,1,1,1,1,'Argent Coat'); -- 2022.03.06 FFXIclopedia and JPWiki agree
```

Comment:
The ingredients (excluding crystals) need to be in order from lower id to higher id. IDK why, but thats what the CI sais.

Reply:
@Xaver-DaRed a quirk of the core code makes the recipes fail if they aren't ordered like that so years ago someone made the sql import fail in that case also so we'd realize when one was set that way. Then later the CI was made to notice that failure also. Someday, I'd like to see a core rework to eliminate the original bug instead of working around it this way. Someday..

Reply:
I noticed they seemed to be ordered didn't know they had to be ^^
My coding is definitely not at the level to help on the core code ^.-

Reply:
@Kayanesa thats ok we don't expect you to try that. Even I don't want to try that one lol.